src/System.Management.Automation/engine/debugger/debugger.cs

```diff
@@ -982,7 +982,7 @@ internal ScriptDebugger(ExecutionContext context)
             _inBreakpoint = false;
             _idToBreakpoint = new ConcurrentDictionary<int, Breakpoint>();
             // The string key is function context file path. The int key is sequencePoint index.
-            _pendingBreakpoints = new ConcurrentDictionary<string, ConcurrentDictionary<int, LineBreakpoint>>();
+            _pendingBreakpoints = new ConcurrentDictionary<string, ConcurrentDictionary<int, LineBreakpoint>>(StringComparer.OrdinalIgnoreCase);
```

Comment:
Should it be only for Windows?

Reply:
Not in this PR. Might be a worth a discussion in a separate issue. There's currently a risk of hitting breakpoints in the wrong file when `file.ps1` and `FILE.ps1` exists in Linux.

This PR simply reverts to original behavior pre 7.4 to fix broken VSCode-extension on Windows. `OrdinalIgnoreCase` is used consistently in the codebase, so a intentional (breaking) change would affect more code, likely outside the debugger as well.

Reply:
Please open tracking issue.

Reply:
Created #20057 for debugger specifically